### PR TITLE
test: speed up tests. clean up some test setup.

### PR DESCRIPTION
**Pull request by pvinis**
This PR resolves []

### Description

⚠️ Please run it on your machines too to confirm the "after" numbers are close to mine. `yarn install`, then `yarn test` should do it. and then run `yarn test` again for a cache variation.

Before:
- 12 sec with cache
- 55 without cache

After:
- 6 with cache
- 20 without cache

Not perfect, but better. I noticed that for our 2 test files, each takes about 2-3 seconds to start, so maybe something in our setupJest is taking long? I tried to even keep that file empty, and still that delay was there, which confused me.

In any case, the tests are a bit faster now, and we can take a look later again.

(I tried vitest quickly, but apparently RN is not yet supported. Hopefully some day soon.)

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [x] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

**Comment by pvinis** [comment]
oh, i also removed the "renders without error" test. its not really a good test, especially with relay not resolving. we basically test that an empty screen will render.

**Comment by pvinis** [comment]
oh god, the conflicts haha. ill fix now.

**Review by rajsam003 — APPROVED** [review]
This looks good to me.
Thanks for this PR @pvinis 🙏🏽

File: src/setupJest.ts [hunk 1]
```diff
@@ -120,18 +32,7 @@ jest.mock("react-native-device-info", () => ({
   hasNotch: jest.fn(),
 }))
 
-const _ = jest.requireActual("lodash")
-
-jest.mock("react-native-reanimated", () => require("react-native-reanimated/mock"))
-
-jest.mock("react-native/Libraries/LayoutAnimation/LayoutAnimation", () => ({
```
Comment (rajsam003) [inline src/setupJest.ts:127]: Why dont we need any of these jest setups?

File: src/Screens/HomeTabs/HomeTabs.tests.tsx [hunk 2]
```diff
@@ -9,17 +8,14 @@ jest.unmock("react-relay")
...
+describe("HomeTabs", () => {
+  const TestRenderer = () => (
+    <RelayEnvironmentProvider environment={mockEnvironment}>
+      <HomeTabs />
+    </RelayEnvironmentProvider>
+  )
 
   it("renders without throwing an error", () => {
-    getWrapper()
+    //     renderWithWrappersTL(<TestRenderer />)
```
[the remark this answers is not in the document]
Comment (pvinis) [reply src/Screens/HomeTabs/HomeTabs.tests.tsx:19]: I wrote a comment in this PR. it's not a good or useful test, so I used it for checking the time, but then commented it out

File: src/setupJest.ts (hunk 1 above)
Comment (pvinis) [reply src/setupJest.ts:127]: it seems we don't need them yet. let's add them once we do need them.

**Comment by rajsam003** [comment]
Lemme run this branch on my end before merging it.

**Comment by rajsam003** [comment]
I am getting this error on my simulator. Do I have to run any command? 

image

**Comment by pvinis** [comment]
thanks for trying it. huh. I guess I fixed the type but didnt assign it?

if you're already there, can you add the first line from https://github.com/artsy/eigen/blob/main/index.ios.js to the index.js file and see if that fixes this?
andnifnit works you can commit it.

**Review by patrinoua — APPROVED** [review]

**Pull request merged**